packages/hdwallet-ledger-webusb/src/adapter.ts

```diff
@@ -24,16 +29,27 @@ export class WebUSBLedgerAdapter {
   private async handleConnectWebUSBLedger(e: USBConnectionEvent): Promise<void> {
     if (e.device.vendorId !== VENDOR_ID) return
 
+    this.connectTimestamp = e.timeStamp
+
+    await timeout(1000) // timeout gives time to detect if it is an app navigation based disconnec/connect event
```

Comment:
Should the number itself go in some `const SCREAMING_GLOBAL = 1000` ?

Reply:
Sure thing lol